ChatSDK/ChatSDKFirebaseAdapter/Classes/Firebase/CCMessageWrapper.m

```diff
@@ -99,7 +99,8 @@ -(NSDictionary *) serialize {
              b_Type: _model.type,
              b_Date: [FIRServerValue timestamp],
              b_UserFirebaseID: _model.userModel.entityID,
-             b_ReadPath: self.initialReadReceipts};
+             b_ReadPath: self.initialReadReceipts,
+             b_Meta: _model.metaDictionary};
```

Comment:
What I meant was that you could just add a check here:

```
b_Meta: _model.metaDictionary ? _model.metaDictionary : @{},
```

Reply:
^^ Review comment above.

Reply:
Fixed